Update size spacing pages (#2249)
jpmorganchase/salt-ds: site/docs/foundations/spacing.mdx

### Headings

Headings provide visual separation and help to establish hierarchy. All headings should be accompanied by standard spacing above and below. The amount of spacing should be based on the following:

- Add `salt-spacing-200` directly above the heading regardless of the level of heading.

> - The spacing placed directly below a heading depends on the level of focus placed on the following item.

Reviewer: Could this do with an example for clarity? I find it a bit abstract.

  label="Show spacing"
/>

Only use separators if the content cannot be effectively divided in another way. Too many repeating lines can overload a design and create unnecessary visual noise.

The [Division](division) Foundation provides further guidance on the use of separators and their alternatives.

<Callout title="Have some feedback?">

> We're still rolling out the Salt Foundations, and we'd appreciate your thoughts and feedback. Please [contact us](../support-and-contributions/index) if you have any comments or suggestions.

Reviewer 2: ```suggestion
The Salt Foundations are still in progress, and we'd appreciate your thoughts and feedback. Please [contact us](../support-and-contributions/index) if you have any comments or questions.
```
